base

```diff
@@ -10,7 +10,7 @@ ROOT_DIR=$(readlink -f $(dirname ${BASH_SOURCE[0]}))
 # Repositories
 REPO=("https://github.com/InfinityGhost/OpenTabletDriver" "$(readlink -f "${ROOT_DIR}/src/OpenTabletDriver")")
 SRC_DIR="${REPO[1]}"
-SRC_DIR_CONFIGURATIONS="${SRC_DIR}/OpenTabletDriver/Configurations"
+SRC_DIR_CONFIGURATIONS="${SRC_DIR}/OpenTabletDriver.Configurations/Configurations"
```

Comment:
Probably remove it all together, we don't need the `Configurations` folder anymore, and will only be used in the future as a way to override in-assembly configurations. I'll update the issue as well to track removal of `Configurations` instead.

Can also just create an empty `Configurations` folder as an alternative.